streaming/pom.xml

```diff
@@ -113,7 +113,7 @@
     </dependency>
     <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
-      <artifactId>htmlunit-driver</artifactId>
```

Comment:
Interesting, we have ui test for streaming?

Reply:
Yes, this one.

https://github.com/apache/spark/blob/master/streaming/src/test/scala/org/apache/spark/streaming/UISeleniumSuite.scala